src/drawing/__tests__/DrawingTarget-test.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { MutableRefObject } from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import DrawingTarget from '../DrawingTarget';
+import { annotations } from '../__mocks__/data';
+import { getShape } from '../drawingUtil';
+
+const { target: mockTarget } = annotations[0];
+
+jest.mock('react', () => ({
+    ...jest.requireActual('react'),
+    useState: jest.fn(),
+}));
+
+describe('DrawingTarget', () => {
+    const defaults = {
```

Comment:
I've come around to the idea that this pattern should be avoided, as `defaults` tends to get referenced in the tests, which leads to sharing of state between tests. We can do a `getDefaults` helper function, instead.